radiomics/base.py

```diff
@@ -63,13 +61,19 @@ def __init__(self, inputImage, inputMask, **kwargs):
       self.logger.warning('Missing input image or mask')
       return
 
+  def _initLesionWiseCalculation(self):
```

Comment:
True, there's no perfect wording that would harmonize with all other systems, but we can define some reasonable terminology and use it consistently in this codebase.